quarkus.ls.ext/com.redhat.quarkus.ls/pom.xml

```diff
@@ -0,0 +1,173 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.redhat.microprofile</groupId>
+	<artifactId>com.redhat.quarkus.ls</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+
+	<name>Quarkus Language Server</name>
+	<description>Quarkus extension for MicroProfile Language Server</description>
+	<url>https://github.com/redhat-developer/quarkus-ls</url>
+	<licenses>
+		<license>
+			<name>EPL-2.0</name>
+			<url>https://www.eclipse.org/legal/epl-2.0/</url>
+			<comments>Eclipse Public License 2.0</comments>
+		</license>
+	</licenses>
+	<scm>
+		<connection>scm:git:git@github.com:redhat-developer/quarkus-ls.git</connection>
+		<developerConnection>scm:git:git@github.com:redhat-developer/quarkus-ls.git</developerConnection>
+		<url>https://github.com/redhat-developer/quarkus-ls</url>
+		<tag>${project.version}</tag>
+	</scm>
+
+	<developers>
+		<developer>
+			<id>team</id>
+			<name>Red Hat Developer</name>
+			<organization>Red Hat</organization>
+			<organizationUrl>https://www.redhat.com</organizationUrl>
+		</developer>
+	</developers>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
+		<dev.build.timestamp>${maven.build.timestamp}</dev.build.timestamp>
+		<lsp4j.version>0.9.0</lsp4j.version>
+		<jboss.releases.repo.id>jboss-releases-repository</jboss.releases.repo.id>
+		<jboss.releases.repo.url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</jboss.releases.repo.url>
+		<jboss.snapshots.repo.id>jboss-snapshots-repository</jboss.snapshots.repo.id>
+		<jboss.snapshots.repo.url>https://repository.jboss.org/nexus/content/repositories/snapshots/</jboss.snapshots.repo.url>
+		<jbossNexus>https://repository.jboss.org</jbossNexus>
+	</properties>
+
+	<build>
+		<resources>
+			<resource>
+				<directory>src/main/resources/</directory>
+				<filtering>true</filtering>
+				<includes>
+					<include>**/*.properties</include>
+				</includes>
+			</resource>
+			<resource>
+				<directory>src/main/resources/</directory>
+				<filtering>false</filtering>
+				<excludes>
+					<exclude>**/*.properties</exclude>
+				</excludes>
+			</resource>
+		</resources>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>build-helper-maven-plugin</artifactId>
+				<version>3.0.0</version>
+				<executions>
+					<execution>
+						<id>parse-version</id>
+						<goals>
+							<goal>parse-version</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
```

Comment:
We don't want to generate a full blown distro here. Might be more interesting to [copy all quarkus dependencies ](https://maven.apache.org/plugins/maven-dependency-plugin/copy-dependencies-mojo.html)(minus the ones already provided by the MP LS) in one directory, those dependencies will then be copied along the quarkus extension into the final server directory.

Reply:
@fbricon updated to use the `maven-jar-plugin` + `maven-dependency-plugin`